test/OpenFeature.Contrib.Providers.Flagd.Test/FlagdConfigTest.cs

```diff
@@ -80,6 +81,80 @@ public void TestFlagdConfigSetCertificatePath()
             Assert.False(config.UseCertificate);
         }
 
+        [Fact]
+        public void TestFlagdConfigFromUriHttp()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("http://domain:8123"));
+
+            Assert.False(config.CacheEnabled);
+            Assert.Equal(new System.Uri("http://domain:8123"), config.GetUri());
+        }
+
+        [Fact]
+        public void TestFlagdConfigFromUriHttps()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("https://domain:8123"));
+
+            Assert.False(config.CacheEnabled);
+            Assert.Equal(new System.Uri("https://domain:8123"), config.GetUri());
+        }
+
+        [Fact]
+        public void TestFlagdConfigFromUriUnix()
+        {
+            CleanEnvVars();
+            var config = new FlagdConfig(new Uri("unix:///var/run/tmp.sock/"));
+
+            Assert.False(config.CacheEnabled);
+            Assert.Equal(new System.Uri("unix:///var/run/tmp.sock/"), config.GetUri());
```

Comment:
nit: I think every use of `System.` in this file is un-needed.

Reply:
Removed :)